macros/edr/hooks/on_run_end.sql

```diff
@@ -1,4 +1,4 @@
 {% macro on_run_end() %}
```

Comment:
I suggested a single on-run-end, based on past feedbacks. @oravi

Reply:
And as we now see, it does not affect functionality.

Reply:
@Maayan-s which feedback? @elongl if you put the first line and "AAAAA" in a macro the entire on-run-end will fail instead of at least trying to run them separately, The handle_test_results is more important than the upload_run_results so we can place the handle_test_results first. Then also when they provide us the log we can know that at least this worked and the upload_run_result didn't

Reply:
_if you put the first line and "AAAAA" in a macro the entire on-run-end will fail instead of at least trying to run them separately._ Hmm, I think I just showed that this is the case for both. The `on-run-end` runs sequentially and it'll stop once there's an error regardless whether it's within the hook's definition or within one of the macros that it invokes.

And I'll move `handle_test_results` to be first.